src/components/generic/generic-table/GenericTable.vue

```diff
@@ -73,12 +73,12 @@
       </table>
       <div
         class="generic-table__container__placeholder"
-        v-if="rows.length === 0 && placeholder"
+        v-if="rows.length === 0"
         :style="{
-          height: `calc(100% - ${rowHeight}px)`
+          height: `calc(100% - ${placeholder ? rowHeight : 0}px)`
         }"
       >
-        <p>{{ placeholder }}</p>
+        <slot name="placeholder">{{ placeholder }}</slot>
```

Comment:
it is, because I added the possibility to fill the placeholder area with html element through slot tag. 
From what I know, it is impossible to identify (within GenericTable) if a slot tag is used by the instanced component. Then we have to let an open door to use it

Reply:
I can't find anything in the doc, but maybe `placeholder` can be removed from props if it is not a prop anymore?

Reply:
It's only in this case that we used the slot tag, for the other cases we still use the placeholder props